rest/data/commit_queue.go

```diff
@@ -88,6 +89,44 @@ func (pc *DBCommitQueueConnector) CommitQueueClearAll() (int, error) {
 	return commitqueue.ClearAllCommitQueues()
 }
 
+func (pc *DBCommitQueueConnector) IsAuthorized(ctx context.Context, username, owner, repo string) (bool, error) {
+	// In the org
+	conf, err := evergreen.GetConfig()
```

Comment:
shouldn't this hang off the environment?

I worry about the global here.

Reply:
done. 
Now passed in by the route